runtime/near-vm-runner/src/wasmtime_runner/logic.rs

```diff
@@ -3925,22 +3933,25 @@ fn read_and_parse_account_id(
     registers: &Registers,
     ptr: u64,
     len: u64,
-) -> Result<AccountId> {
+    config: &Config,
+) -> Result<AccountId, VMLogicError> {
```

Comment:
why did you need to specify `VMLogicError` explicitly here? the original shorthand should work: `Result<T, E = VMLogicError>`